CHANGELOG.md

```diff
@@ -13,6 +14,9 @@
 
 * Now supports case insensitive queries for UWP.
 * Upgraded Visual Studio project to version 2017.
+* Enable reading and writing of big blobs via Table interface.
+  Only to be used by Sync. The old interface still has a check on 
```

Comment:
If you put in some data that is bigger than the limit, get_binary cannot return the data, and will just return null. This also means that queries will not work.